apps/experiments/views/experiment.py

```diff
@@ -232,6 +235,12 @@ def __init__(self, request, *args, **kwargs):
         self.fields["synthetic_voice"].widget.template_name = "django/forms/widgets/select_dynamic.html"
         self.fields["llm"].widget.template_name = "django/forms/widgets/select_dynamic.html"
 
+    def clean_participant_whitelist(self):
+        cleaned_identifiers = []
+        for identifier in self.cleaned_data["participant_whitelist"]:
+            cleaned_identifiers.append(identifier.replace(" ", ""))
```

Comment:
add a validation error if not in E164 format: (can use `re.compile(r'^\+[1-9]\d{1,14}$')`) (based on https://www.twilio.com/docs/glossary/what-e164)

Reply:
Unfortunately we can't validate it like that since it depends on the experiment channels. But perhaps we could use the configured channels to do the validation?

Reply:
Is the suggestion that we take the input list and run it though all configured experiment channel validators (that I need ot introduce). If it fails all of them, we know it's an invalid identifier. If it passes at least one of them, we know it is valid. Is this what ya'll is suggesting?

Reply:
yes, but I'm not convinced it's worth it yet

Reply:
@stephherbers are you fine with this not going in right now?